CMakeLists.txt

```diff
@@ -126,6 +126,8 @@ install(FILES ${CMAKE_CURRENT_SOURCE_DIR}/LICENSE
 #--- project specific subdirectories -------------------------------------------
 add_subdirectory(python)
 add_subdirectory(src)
+
+option(BUILD_TESTING "Enable and build tests" ON)
```

Comment:
This option is already set by `include(CTest)` https://cmake.org/cmake/help/v3.12/module/CTest.html
If this is just have the option explicitly in the File it should be moved to before the `include(CTest)`, so that at least its default value would be used if someone where to change it.

Reply:
Ah right. I didn't know that. In that case this is completely unnecessary and I will close it. Sorry for the noise.